test/transaction.ts

```diff
@@ -163,7 +171,7 @@ describe('Transaction', () => {
     it('should accept gaxOptions', done => {
       const gaxOptions = {};
 
-      transaction.request_ = config => {
+      transaction.request_ = (config: RequestConfig) => {
```

Comment:
Here and below - we want `transaction` to be properly typed, as called out above.  That way, you can change this to `sandbox.stub` and have it retain typing for you, so we don't need to say the type twice.